share/telemetry.py

```diff
@@ -85,18 +85,20 @@ class TelemetryData:
     with_exception: Optional[WithExceptionTelemetryEnum] = None
     to_be_continued: bool = False
 
-    input_outputs_type: dict[str, list[str]] = {}
+    input_outputs_type: dict[str, dict[str, str]] = {}
     input_is_continuing: dict[str, bool] = {}
     events_forwarded: dict[str, int] = {"sent": 0, "empty": 0, "skipped": 0}
 
     output_sent_to_replay: dict[str, int] = {}
 
-    def add_output_type_for_input(self, input_id: str, output_type: str) -> None:
+    def set_output_type_for_input(self, input_id: str, input_type: str, output_type: str) -> None:
         """Add the output type for the input."""
         if input_id not in self.input_outputs_type:
-            self.input_outputs_type[input_id] = list()
-
-        self.input_outputs_type[input_id].append(output_type)
+            self.input_outputs_type[input_id] = {
```

Comment:
while we have only one "elasticsearch" output at the moment, we are going soon to add also the logstash one.
potentially every input can have as many outputs as the number of types we support.
this change seems to me that reduce to a single one the data related to the output for a given input

Reply:
Oh, I forgot the Logstash output is coming 🤦  Thank you for the heads up!